piet-wgsl/src/shaders/preprocess.rs

```diff
@@ -7,24 +7,26 @@ use std::{
 
 pub fn get_imports(shader_dir: &Path) -> HashMap<String, String> {
     let mut imports = HashMap::new();
-    let imports_dir = shader_dir.join("shared");
-    for entry in imports_dir
-        .read_dir()
-        .expect("Can read shader import directory")
+    // When targeting WASM, we need to directly bundle shader source strings at compile-time since
+    // we cannot read from the local build directory at runtime.
```

Comment:
This justification is true, however I'm not sure I follow how the inference was made. Another viable option may be to host the shaders on the web server, and then `fetch` them at runtime. That alternative doesn't appear to have been recognised here.

I think including them verbatim is probably the right tradeoff either way; however I suspect that will be the end state on all platforms, not just native.

Although at some point we should probably allow hot reload of the shaders for development (although these aren't exactly the kind of shaders that work very well with hot reload, it can't hurt either).